packages/core/saved-objects/core-saved-objects-server/src/import.ts

```diff
@@ -69,6 +69,14 @@ export interface SavedObjectsImportOptions {
    * different Kibana versions (e.g. generate legacy URL aliases for all imported objects that have to change IDs).
    */
   compatibilityMode?: boolean;
+  /**
+   * If true, will import as a managed object, else will import as not managed.
```

Comment:
> If true, will import as a managed object, else will import as not managed.

This description seems false. From the my understanding of the code:
- if `true` => all imported objects will have `managed: true`
- if `false` => all imported objects will have `managed: false`
- if unset => all imported objects will retain the value their currently had for `managed`, defaulting to `false` if it was unset.

Or did I misunderstood the code?

Reply:
You understand the code correctly and that's what the comment implies (rather poorly it seems 😓 )